build/bin/build-client-binary

```diff
@@ -24,6 +24,8 @@ gobuild_component() {
     fi
     if [ ${component} = 'pmm-admin' -o ${component} = 'pmm-agent' ]; then
         build_command="cd /tmp/go/src/${extract_path} && make release"
+    elif [ ${component} = 'mongodb_exporter' ]; then
+        build_command="cd /tmp/go/src/${extract_path} && make build"
```

Comment:
@percona-csalguero @EvgeniyPatlan any objections to rename that to `make release` just for consistency?

Reply:
Ah, we have release already – for Go Releaser. Please ignore that